README.md

```diff
@@ -1,168 +1,8 @@
-<!-- BEGIN MODULE HOOK -->
+<!-- BEGIN TESTS HOOK -->
```

Comment:
README is empty, removed too much. This needs to be our standard readme including hooks for pre-commit documentation.